go/plugins/vertexai/vertexai.go

```diff
@@ -70,7 +71,19 @@ func Init(ctx context.Context, projectID, location string) error {
 	if state.initted {
```

Comment:
Update func comment three lines up from here to clarify when you need to call DefineModel.

Also for consistency shouldn't we define all known embedding models?